packages/stylelint-plugin-ckeditor5-rules/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
`jest` and other dependencies required for testing purposes should be mentioned as dev-deps.